tests/integration/olp-cpp-sdk-dataservice-read/CatalogClientTest.cpp

```diff
@@ -100,25 +100,6 @@ TEST_P(CatalogClientTest, GetCatalog403) {
   ASSERT_EQ(403, catalog_response.GetError().GetHttpStatusCode());
 }
 
-TEST_P(CatalogClientTest, GetPartitions) {
```

Comment:
Should we remove the `CatalogClient` tests? I guess we have (or should have) separate ticket for that.

Reply:
There is a requirement to move the tests.